addons/website/static/src/components/media_dialog/image_selector.js

```diff
@@ -1,13 +1,106 @@
 /** @odoo-module **/
 
+import { ConfirmationDialog } from "@web/core/confirmation_dialog/confirmation_dialog";
+import { _t } from "@web/core/l10n/translation";
 import { patch } from '@web/core/utils/patch';
-import { ImageSelector } from '@web_editor/components/media_dialog/image_selector';
+import { sprintf } from "@web/core/utils/strings";
+import { AutoResizeImage, ImageSelector } from "@web_editor/components/media_dialog/image_selector";
+import { AttachmentError } from "@web_editor/components/media_dialog/file_selector";
+import { markup } from "@odoo/owl";
+
+patch(AutoResizeImage.prototype, "image_attachment_website", {
+    /**
+     * @override
+     */
+    async remove() {
+        const ids = await this.props.getOriginalAndCopiesIds(this.props.id);
+        const dialogBody = markup(sprintf(`%s
```

Comment:
The dialog body takes a string passed to a `t-out` in the template. `markup` is thus the only possibility to add a checkbox there.